AmplifyPlugins/Core/AWSPluginsCore/Model/Support/QueryPredicate+GraphQL.swift

```diff
@@ -47,11 +69,26 @@ public struct GraphQLFilterConverter {
 /// Extension to translate a `QueryPredicate` into a GraphQL query variables object
 extension QueryPredicate {
 
+    public func graphQLFilter(for modelSchema: ModelSchema?) -> GraphQLFilter {
+        Amplify.log.warn("\(#function) is for internal use only")
```

Comment:
1.  Remove this warning statement
2. Add a comment to the effect of:
```
/// - Warning: Although this has `public` access, it is intended for internal use and should not be used directly
///   by host applications. The behavior of this may change without warning.
```
(search code to copy and paste)